Explorer/Assets/DCL/MapPins/Systems/MapPinLoaderSystem.cs

```diff
@@ -22,18 +23,22 @@ namespace DCL.SDKComponents.MapPins.Systems
     [UpdateInGroup(typeof(ComponentInstantiationGroup))]
     public partial class MapPinLoaderSystem : BaseUnityLoopSystem
     {
+        private const int ATTEMPTS_COUNT = 6;
+
         private readonly World globalWorld;
         private readonly IPartitionComponent partitionComponent;
-        private const int ATTEMPTS_COUNT = 6;
         private readonly ISceneData sceneData;
+        private readonly bool useCustomMapPinIcons;
+
         private int xRounded;
         private int yRounded;
 
-        public MapPinLoaderSystem(World world, ISceneData sceneData, World globalWorld, IPartitionComponent partitionComponent) : base(world)
+        public MapPinLoaderSystem(World world, ISceneData sceneData, World globalWorld, IPartitionComponent partitionComponent, FeatureFlagsCache featureFlagsCache) : base(world)
         {
             this.sceneData = sceneData;
             this.globalWorld = globalWorld;
             this.partitionComponent = partitionComponent;
+            useCustomMapPinIcons = featureFlagsCache.Configuration.IsEnabled("alfa-map-pins-custom-icons");
```

Comment:
Maybe we could extract `"alfa-map-pins-custom-icons"` into a constant?

Reply:
Yeah, I have done it in the Portable Experiences branch and I didnt want to do it again here because of conflicts, once I merge this, I will add this to the constant's list on the PX branch :D